tests/bplan/test_bplan_api.py

```diff
@@ -7,6 +9,7 @@
 from adhocracy4.modules import models as module_models
 from adhocracy4.phases import models as phase_models
 from meinberlin.apps.bplan import models as bplan_models
```

Comment:
as you import the Bplan directly below, can you remove this import and adjust the code accordingly.

Reply:
ah, yes, nice catch